torch/_inductor/lowering.py

```diff
@@ -3630,6 +3631,16 @@ def sym_numel(a):
     return a.get_numel()
 
 
+@register_lowering(sym_float)
+def sym_float(a):
+    return a
+
+
+@register_lowering(sym_floor)
+def sym_floor(a):
+    return a
```

Comment:
this lowering looks wrong if you have a symfloat lol

Reply:
Could you explain in more detail what needs to be done instead? I basically copied what `sym_float` does.

Reply:
If I pass 4.5 into this function, it needs to go to 4, but right now this noops